python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -106,6 +106,8 @@ def __statisticsSummary( plug ) :
 	scene. These should be respected by all renderers.
 	""",
 
+	## For some of these tooltips, we hackily grab the metadata from
```

Comment:
We shouldn't be 'hackily' doing anything here.

The pattern in `CameraTweaksUI` that interrogates the metadata system can be used, or if we are sharing the dict instead, we shouldn't be relying on any order-dependence. You simply `import GafferSceneUI.CameraUI` at the top if you want to make use of it. This works even whilst the module is importing as you're actually just directly asking it to load the child module. If you tried to use `import GafferSceneUI` and access it from there that would then become order dependent.

Reply:
Checked with @johnhaddon, we should use the shared dict approach over interrogating metadata....